exporter/collector/integrationtest/testdata/fixtures/workload_metrics_expect.json

```diff
@@ -6454,7 +6424,7 @@
                   "startTime": "1970-01-01T00:00:00Z"
                 },
                 "value": {
-                  "int64Value": "203"
```

Comment:
Note: this is a change in the self-obs metrics, since we now send 202 metrics instead of 203 metrics.